### ISSUE TITLE
Training and testing on same data

### ISSUE THREAD

**Issue by saurabhrathor** [Naresh1318/GenderClassifier#1]
Hi Naresh,
You are using the same dataset for training and testing. Will it not create an Overfit Classifier and give higher accuracy?

**Comment by Naresh1318** [comment]
Ideally yes, you aren't supposed to use the trained data for evaluating the test accuracy. But the video didn't provide any additional test data and Siraj had done it using the same training data so I didn't bother to change it.
